### DISCUSSION TITLE
Improvements in browsing the list

### DISCUSSION THREAD

**Discussion by goyalyashpal** [Klocman/Bulk-Crap-Uninstaller#596] [Ideas]
### querying (select, sort, groupby)

* Some ability to view list of selected ones 
* group by some property and collapse
* multi-column sort

### viewing

* toggle full width vs last specified
* hover to preview the column header 
useful for cols whose values are smaller (Yes/No), but header name itself is long (Protected, System, Startup, Uninstaller Kind)

**Comment by Klocman** [comment]
Thanks for the suggestions! It would help if there was a visual mock-up representation of some of those points.

**Reply by goyalyashpal** [reply]
> It would help if there was a visual mock-up representation of some of those points

i agree, but unfortunately, i don't know how to create mock-ups effectively... 

do u have any recommendations for some application for that?  in the best case it should be local FOSS, followed by true gratis local, followed by true gratis online in worst case...  so, figma etc are out of the equation.

**Reply by Klocman** [reply]
Even mspaint would work, honestly.

**Comment by Klocman** [comment]
Closing since no details were provided.
